pkg/apis/core/v1alpha1/federationconfig_types.go

```diff
@@ -17,13 +17,14 @@ limitations under the License.
 package v1alpha1
 
 import (
+	apiextv1b1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // FederationConfigSpec defines the desired state of FederationConfig
 type FederationConfigSpec struct {
 	// Whether the federation namespace will be the only target for federation.
```

Comment:
-> The scope of the federation control plane (`Namespaced` or `Cluster`)

Reply:
Yes. I use `scope` with value `Namesapced` or `Cluster` for namespaced federation and cluster scoped federation. This is similar like `CRD` definitions in using `scope`

Reply:
I get that.  I'm requesting that you update the now-invalid comment on line 26 to reflect the new field.

Reply:
done